setup.py

```diff
@@ -15,7 +15,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # http://packaging.python.org/en/latest/tutorial.html#version
-    version='0.12.2',
+    version='0.13.0',
```

Comment:
Since this is dropping support for previous APIs, it should probably be a major version update, not minor.

Reply:
Good catch @ThrawnCA ! I'm fixing this and proposing a `1.0.0` version.